libraries/libfc/test/crypto/test_utils.hpp

```diff
@@ -1,9 +1,9 @@
-uint32_t to_uint32(const std::string& s) {
+inline uint32_t to_uint32(const std::string& s) {
```

Comment:
Just our of curiosity, did you need to add `inline` here for a specific reason?

Reply:
Yes, it is included in 3 of the test files.

Reply:
Oh, I get it. I probably would have added `static`. I didn't think that `inline` changed the linkage status (external) for non-member functions.
Actually, I don't believe it does, and I'm surprised that adding `inline` fixed the issue.

Reply:
It is literally the only use of `inline` these days. Compilers typically ignore `inline` for determining if a function should be inlined.
https://en.cppreference.com/w/cpp/language/inline

Reply:
Yes, it is somewhat equivalent now.  Do you feel there is a reason to prefer `inline` to `static`? It may be moot in this case, but if the functions are not actually inlined,  `static` and `inline` would make a difference.

Reply:
Maybe more of a preference. https://stackoverflow.com/questions/10876930/should-one-never-use-static-inline-function I stopped using `static` years ago but can't remember the reason why.

Reply:
I believe that, for the case where the compiler does *not* inline the functions, `static` would cause each compilation unit to have a private copy of the function, while `inline` would cause each compilation unit to export a function with external linkage, which would then later be merged into one by the linker.
`static`: larger code size, less work for linker, maybe better performance as the code may be closer to where it is used 
`inline`: the opposite of above